### ISSUE TITLE
Manage Cancellation Options Idea ?

### ISSUE THREAD

**Issue by spirogg** [PHPfox-Official/phpfox-v4-feature-requests#742]
Dear friends,

Manage Cancellation Options Idea ?

how can we make this area better to stop cancellations?

maybe when they click on a link to cancel we can have a popup for each cancellation that asks if there is something more we can do to keep them happy ? 

- example: 
**I have a privacy concern**

when the user selects that option: we can have a **popup with a link to the /user/privacy/ section**, 
the popup would say: _have you tried setting up your privacy settings?_ _if so and you still want to cancel, we are sorry to see you go_

"**Continue to Cancel**"

--------------------------------------------------------------------------------------------------------------

- or another example:

If they want to cancel because they choose: **I'm getting too much email from this site**

when the user selects that option: we can have a **popup with a link to the /user/privacy/ email notifications section**, 
the popup would say: _have you tried not allowing email notifications in your privacy settings?_ _if so and you still want to cancel, we are sorry to see you go_

"**Continue to Cancel**"

---------------------------------------------------------------------------------------------------------------

- or another example:

I don't understand how to use this site

when the user selects that option: we can have a **popup with a link to KB of how to use the site**,

the popup would say: _have you tried Knowledge Base?_ _if so and you still want to cancel, we are sorry to see you go_

"**Continue to Cancel**"

-----------------------------------------------------------------------------------------------------

- so basically as an admin we can have an option to turn this feature on or off for Manage Cancellations

the popup template should be the same except for an area we can fill in and give us an area we can add a link to help users.

- **admin Template for popup** 

**when adding or when editing a reason**, in _settings_ we can have an area to add wording for each one and be able to add links: where we want a user to click and a set button that says 
"**Continue to Cancel**"

- see photo below here of my idea how to set this up?

- just an Idea not sure if this is possible

![template idea](https://user-images.githubusercontent.com/17736688/47598611-c7179000-d963-11e8-9e08-1935600d5b02.png)

-------------------------------------------------------------------------------------------
